unicodetools/src/main/java/org/unicode/props/PropertyParsingInfo.java

```diff
@@ -674,6 +686,20 @@ static void parseSourceFile(
                                         ? null
                                         : nextProperties.getProperty(propInfo.property));
                     } else {
+                        if (fileName.equals("math/*/MathClassEx")
+                                && indexUnicodeProperties.ucdVersion.compareTo(
+                                                VersionInfo.UNICODE_6_3)
+                                        <= 0) {
+                            parser =
+                                    parser.withLinePreprocessor(
+                                            s ->
+                                                    s.startsWith("FE61-FE68;")
```

Comment:
Same here. Ideally people shouldn't be using two different syntaxes for ranges, but is it worth matching a pattern rather than a string?